apps/ima/views/declaration.html

```diff
@@ -1,9 +1,9 @@
+<title>{{#t}}pages.declaration.header{{/t}} – GOV.UK</title>
 {{<partials-page}} {{$page-content}}
  {{#markdown}}declaration{{/markdown}}
  <br><br>
   {{#renderField}}person-declaration{{/renderField}} 
   <br><br>
-  <br><br>
   <p>You will not be able to change your answers after you submit.</p>
   <br><br>
```

Comment:
Don't think you need these break tags, the gaps are too wide compared to the figma.

Reply:
Break tags removed